src/utils/settingUtils.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+
+const extensionPrefix: string = "azureTerraform";
+
+export function isTerminalSetToCloudShell(): boolean {
+    return getTerraformExtensionSetting<string>("terminal") === "cloudshell";
+}
+
+export function getSyncFileBlobPattern(): vscode.GlobPattern {
+    return getTerraformExtensionSetting<vscode.GlobPattern>("files");
+}
+
+export async function updateGlobalSetting<T = string>(section: string, value: T): Promise<void> {
```

Comment:
suggest to remove `updateGlobalSetting` and `updateWorkspaceSetting` in this PR because they're not used anywhere. And add them later on demand.

Reply:
ok